client/src/pages/About/About.jsx

```diff
@@ -1,12 +1,71 @@
 import React from 'react'
 import './About.css'
+import flowers3 from '../../assets/Flowers3.png'
+import flowers2 from '../../assets/Flowers2.png'
+import treesBig from '../../assets/TreesBig.png'
+import treesSmall from '../../assets/TreesSmall.png'
+import ladyLaptop from '../../assets/LadyLaptop.png'
+import manLaptop from '../../assets/ManLaptop.png'
 
 const About = () => {
     return (
         <div className="page-content">
-            <h1>About us</h1>
+            <img src={flowers3} alt = "three flowers" 
+                style = {{position: 'relative', left: 260, top: 113, height: '3.187%', width: '3.187%'}}
+            />
+            <img src={flowers2} alt = "two flowers"
+                style = {{position: 'relative', left: 715, top: 45, height: '3.187%', width: '3.187%'}}
+            />
+            <img src={treesBig} alt = "big trees"
+                style = {{position: 'relative', left: 565, top: 370, height: '12%', width: '12%'}}
+            />
+            <img src={treesSmall} alt = "small trees"
+                style = {{position: 'relative', left: 1000, top: 388, height: '9%', width: '9%'}}
+            />
+            <img src={ladyLaptop} alt = "lady laptop"
+                style = {{position: 'relative', left: 490, top: 310, height: '9.1%', width: '9.1%'}}
+            />
+            <img src={manLaptop} alt = "man laptop"
+                style = {{position: 'relative', left: 505, top: 291, height: '12%', width: '12%'}}
+            />
+            <style>
+                @import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
+            </style>
```

Comment:
Can you move this line to `/client/src/App.css`? That's where global css lives, and chances are that this font will be used on a lot of other pages too.

Reply:
Updated!